GLOSSARY.md

```diff
@@ -0,0 +1,53 @@
+## Glossary
+
+Before getting started, we recommend familiarizing yourslef with the following terms and their definitions within the Shorebird ecosystem.
```

Comment:
yourself

Reply:
Although, honestly I'd remove this.  I'm not sure folks read these things first. 🤣  Some do I'm sure. 🤷

Reply:
Yeah good catch just removed it for now as you suggested